### PR TITLE
Eventhub template

### PR DESCRIPTION
**Pull request by rokatyy**
Some of the templates generated from the https://github.com/nuclio/nuclio/tree/development/hack/examples in the `generated.go` file are not very useful because they require manual modifications of the YAML file itself. For example, the "eventhub:golang" template includes numerous strings such as namespace: `< your value here >`, making it impossible to utilize from the UI. We aim to enhance user-friendliness by relocating it to the this repository, allowing users to specify all parameters via the UI.
 

Jira - https://jira.iguazeng.com/browse/NUC-135

### CODE REVIEW

File: eventhub/function.yaml.template [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+spec:
+  description: >
+    An Azure Event Hub triggered function with a configuration that connects to an Azure Event Hub.
+    The function reads messages from two partitions, process the messages, invokes another function,
+    and sends the processed payload to another Azure Event Hub.
+
+  # Note that we're not specifying handler. This is because the Golang runtime
+  # can find handlers automatically.
+  runtime: "golang"
+  handler: main:SensorHandler
```
Comment (TomerShor) [inline eventhub/function.yaml.template:14]: But we are specifying the handler

File: eventhub/function.yaml.template [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        sharedAccessKeyName: {{ .SharedAccessKeyName }}
+        sharedAccessKeyValue: {{ .SharedAccessKeyValue }}
+        namespace: {{ .Namespace }}
+        eventHubName: {{ .EventHubName }}
+        consumerGroup: {{ .ConsumerGroup }}
+         # Match event hub partition  number
+        partitions:
+        - 0
+        - 1
+   # Output to another event hub
```
Comment (TomerShor) [inline eventhub/function.yaml.template:35]:
```suggestion
  # Output to another event hub
```

**Review by TomerShor — CHANGES_REQUESTED** [review]
Really minor

File: eventhub/function.yaml.template (hunk 1 above)
Comment (rokatyy) [reply eventhub/function.yaml.template:14]: @TomerShor what do you mean? The name of handler in the code is `SensorHandler`

Comment (TomerShor) [reply eventhub/function.yaml.template:14]: That's fine, naming doesn't matter.
but the comment says "we're not specifying handler", and 2 rows below there is a specified handler:
```
handler: main:SensorHandler
```

Comment (rokatyy) [reply eventhub/function.yaml.template:14]: @TomerShor ah, now I noticed it :) will remove the comment

**Review by TomerShor — APPROVED** [review]

**Pull request merged**